versions.yaml

```diff
@@ -278,7 +278,10 @@ externals:
       package_output_dir: "OvmfX64"
     sev:
       description: "AmdSev build needed for SEV measured direct boot."
-      version: "edk2-stable202202"
+      version: "edk2-202208-sev-alloc-runtime-secret-area"
+      # CCv0 switching to this fork until new release of edk2 has fix
```

Comment:
Please, fix the comment to not refer to the CCv0 branch.

Reply:
Removed.